app_server/app/services/UserService.js

```diff
@@ -49,6 +49,20 @@ Class.getUserById = function (id) {
   });
 };
 
+Class.getListOfUsers = function (filters) {
+  logger.debug('Getting list of users with filters: %j', filters);
+
+  return Storage.getListOfUsers(filters)
+  .then(function receiveResult(result) {
+    if (result) {
+      return result.map(function(res) { return res.dataValues; });
```

Comment:
Your choice to make, but if we're allowing ES6, you can write the inside part as `(res) => res.dataValues` (initial brackets optional if only 1 parameter).